app/assets/stylesheets/variables/_app.scss

```diff
@@ -22,15 +22,15 @@ $line-height-2: 1.3 !default;
 $line-height-3: 1.75 !default;
 $line-height-4: 2 !default;
 
-$h1: 1.75rem !default;
-$h2: 1.5rem !default;
-$h3: 1.125rem !default;
+$h1: 1.625rem !default;
+$h2: 1.25rem !default;
+$h3: 1.0rem !default;
```

Comment:
Looks like the sass linter caught you here

Reply:
This was fixed.  It was due to the 1.0 instead of 1.  The linter wanted it to have no mantissa.  This is corrected.

Reply:
This should be ready for merge, pending a review.